perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/registrartabs/CreateFormItemTabItem.java

```diff
@@ -79,6 +79,8 @@ public class CreateFormItemTabItem implements TabItem {
 		aMap.put("HEADING", "Header");
 		aMap.put("TIMEZONE", "Selection of timezone");
 		aMap.put("EMBEDDED_GROUP_APPLICATION", "Groups for embedded applications");
+		aMap.put("LIST_INPUT_BOX", "Field used to gather attribute list values");
+		aMap.put("MAP_INPUT_BOX", "Field used to gather attribute value in form of key - value pairs");
```

Comment:
I probably wouldn't use the same text for the "name" (here) and "description". So maybe something similar to texts above:
```java
		aMap.put("LIST_INPUT_BOX", "Multiple input text fields");
		aMap.put("MAP_INPUT_BOX", "Multiple key-value input text fields");
```

Reply:
More appropriate name was used.